source/GameInterface.Tests/Serialization/SerializerTests/ItemRosterElementSerializationTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using GameInterface.Serialization.Impl;
+using GameInterface.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TaleWorlds.CampaignSystem.Roster;
+using Xunit;
+using TaleWorlds.Core;
+using System.Reflection;
+
+namespace GameInterface.Tests.Serialization.SerializerTests
+{
+    public class ItemRosterElementSerializationTest
+    {
+        [Fact]
+        public void ItemRosterElement_Serialize()
+        {
+            ItemRosterElement itemRosterElement = new ItemRosterElement();
+
+            BinaryPackageFactory factory = new BinaryPackageFactory();
+            ItemRosterElementBinaryPackage package = new ItemRosterElementBinaryPackage(itemRosterElement, factory);
+
+            package.Pack();
+
+            byte[] bytes = BinaryFormatterSerializer.Serialize(package);
+
+            Assert.NotEmpty(bytes);
+        }
+
+        [Fact]
+        public void ItemRosterElement_Full_Serialization()
+        {
+            ItemRosterElement itemRosterElement = new ItemRosterElement();
+            FieldInfo _amount = typeof(ItemRosterElement).GetField("_amount",BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Public);
```

Comment:
Update this to be similar to https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/taverns-serialization/source/GameInterface.Tests/Serialization/SerializerTests/CharacterSkillsSerializationTest.cs#L29